src/screen/RequestScreen.js

```diff
@@ -238,6 +235,18 @@ class RequestScreen extends Screen {
 	}
 
 	/**
+	 * Adds aditional data to the body of the request in case a submit button
+	 * is captured during form submission.
+	 * @param {!FormData} body The FormData containing the request body.
+	 */
+	maybeAppendSubmitButtonValue(body) {
+		const button = globals.capturedFormButtonElement;
+		if (body && button && button.name && !button.disabled) {
```

Comment:
Why are you checking if the button is not disabled here? Once the disabled button prevents the form submission the screen will not be loaded. Right?